### PR TITLE
Seperating API from builder and SIGTERM

### PR DESCRIPTION
**Pull request by viennaa**
it doesn't belong together and should have been separated from the beginning already
SIGTERM handling added as a prerequisite for upcoming features

### CODE REVIEW

File: InventoryBuilder.py [hunk 1]
```diff
@@ -1,25 +1,17 @@
-import urllib3.exceptions
-from flask import Flask
-from gevent.pywsgi import WSGIServer
-from threading import Thread
 from tools.Vrops import Vrops
 from tools.helper import yaml_read
 from collections import defaultdict
+from tools.InventoryApi import InventoryApi
```
Comment (richardtief) [inline InventoryBuilder.py:4]: I'm not sure if `InventoryApi` is a tool. I would prefer to group both inventory parts in one folder. e.g. `/inventory`

**Review by richardtief — COMMENTED** [review]
Looks good. Just one little remark and one fix which I pushed directly.

File: InventoryBuilder.py (hunk 1 above)
Comment (viennaa) [reply InventoryBuilder.py:4]: Valid point, it deserves its own directory now.

**Review by richardtief — APPROVED** [review]
Great. Thanks!

**Pull request merged**